packages/terra-application-layout/examples/index-examples/ExampleApplication.jsx

```diff
@@ -0,0 +1,319 @@
+    '/page_2': {
+      path: '/page_2',
+      component: {
+        default: {
+          componentClass: ApplicationContent,
+          props: {
+            basePath: '/page_2',
+            contentName: 'Page 2',
+            noMenu: true,
+          },
+        },
+      },
+    },
+    '/page_3': {
+      path: '/page_3',
+      component: {
+        default: {
+          componentClass: ApplicationContent,
+          props: {
+            basePath: '/page_3',
+            contentName: 'Page 3',
+            showDummyContent: true,
+          },
+        },
+      },
+    },
+    '/page_4': {
+      path: '/page_4',
+      component: {
+        default: {
+          componentClass: ApplicationContent,
+          props: {
+            basePath: '/page_4',
+            contentName: 'Page 4',
+            showDummyContent: true,
+          },
+        },
+      },
+    },
+    '/page_5': {
+      path: '/page_5',
+      component: {
+        default: {
+          componentClass: ApplicationContent,
+          props: {
+            basePath: '/page_5',
+            contentName: 'Page 5',
+            showDummyContent: true,
+          },
+        },
+      },
+    },
+    '/page_6': {
+      path: '/page_6',
+      component: {
+        default: {
+          componentClass: ApplicationContent,
+          props: {
+            basePath: '/page_6',
+            contentName: 'Page 6',
+            showDummyContent: true,
+          },
+        },
+      },
+    },
+    '/page_7': {
+      path: '/page_7',
+      component: {
+        default: {
+          componentClass: ApplicationContent,
+          props: {
+            basePath: '/page_7',
+            contentName: 'Page 7',
+            showDummyContent: true,
+          },
+        },
+      },
+    },
+  },
+};
+
+/**
+ * The navigationItems will be used to present the ApplicationLayout's navigation controls. The paths provided here must be present in
+ * the routingConfig. If no navigation controls are desired, these items can be omitted.
+ *
+ * With standard rendering, the items will be presented as tabs within the ApplicationLayout's header.
+ * With compact rendering, the items will be presented within the layout's menu region within a ApplicationLayout-managed menu.
+ */
+const navigationItems = [{
+  path: '/page_1',
+  text: 'Page 1',
+}, {
+  path: '/page_2',
+  text: 'Page 2',
+}, {
+  path: '/page_3',
+  text: 'Page 3',
+}, {
+  path: '/page_4',
+  text: 'Page 4',
+}, {
+  path: '/page_5',
+  text: 'Page 5',
+}, {
+  path: '/page_6',
+  text: 'Page 6',
+}, {
+  path: '/page_7',
+  text: 'Page 7',
+}];
+
+/**
+ * The indexPath will be given to the NavigationLayout to set up the appropriate redirects. If users attempt to navigate to a path unsupported
+ * by the routingConfig, they will be redirected to this route. This path should therefore be present in the routingConfig.
+ */
+const indexPath = '/page_1';
+
+const userAvatar = (
+  <Avatar
+    // image={ProfilePicture}
```

Comment:
Can this be uncommented?

Reply:
Yep, the Image has a little bug that causes some flickering when re-rendered. I commented this out to see if it'd look better without the flicker. It does, but I like Kurt Russell too much, so I put it back in.

Reply:
![](https://media.giphy.com/media/xnvXXXbNxjxok/giphy.gif)